app/views/dashboard/protocols/form/study_form_sections/_interactive_form.html.haml

```diff
@@ -20,19 +20,20 @@
 
 
 .form-group.row
-  = form.label :selected_for_epic,
-    t(:protocols)[:studies][:information][:push_to_epic],
-    class: 'col-lg-2 control-label required',
-    data: { toggle: 'tooltip', placement: 'right', delay: '{"show":"500"}' },
-    title: t(:protocols)[:tooltips][:epic]
-  .col-lg-10
-    .btn-group#selected_for_epic_button{data: {toggle: "labels"}}
-      %label.radio-inline.btn.btn-default#study_selected_for_epic_true_button{ class: protocol.selected_for_epic? ? 'active' : ''}
-        = form.radio_button :selected_for_epic, 'true', id: "study_selected_for_epic_true", class: 'hidden'
-        = t(:constants)[:yes_select]
-      %label.radio-inline.btn.btn-default#study_selected_for_epic_false_button{ class: protocol.selected_for_epic == false ? 'active' : ''}
-        = form.radio_button :selected_for_epic, 'false', id: "study_selected_for_epic_false", class: 'hidden'
-        = t(:constants)[:no_select]
+  - if USE_EPIC
+    = form.label :selected_for_epic,
+      t(:protocols)[:studies][:information][:push_to_epic],
+      class: 'col-lg-2 control-label required',
+      data: { toggle: 'tooltip', placement: 'right', delay: '{"show":"500"}' },
+      title: t(:protocols)[:tooltips][:epic]
+    .col-lg-10
+      .btn-group#selected_for_epic_button{data: {toggle: "labels"}}
+        %label.radio-inline.btn.btn-default#study_selected_for_epic_true_button{ class: protocol.selected_for_epic? ? 'active' : ''}
+          = form.radio_button :selected_for_epic, 'true', id: "study_selected_for_epic_true", class: 'hidden'
+          = t(:constants)[:yes_select]
+        %label.radio-inline.btn.btn-default#study_selected_for_epic_false_button{ class: protocol.selected_for_epic == false ? 'active' : ''}
+          = form.radio_button :selected_for_epic, 'false', id: "study_selected_for_epic_false", class: 'hidden'
+          = t(:constants)[:no_select]
 .form-group.row.selected_for_epic_dependent{ display_if(protocol.selected_for_epic) }
```

Comment:
Should these `.selected_for_epic_dependent`s also be inside of the `-if USE_EPIC` block?